angular-cli/.vscode/extensions.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
did you intentionally commit these .vscode files?

Reply:
Just curious, I don't mind keeping them since most devs probably use vscode anyway